### DISCUSSION TITLE
Problem downloading specific feed version GTFS zip via v2 REST API

### DISCUSSION THREAD

**Discussion by shiqin-liu** [transitland/transitland#351] [Q&A] [answered]
Hi! I try to download the a specific feed version GTFS zip with feed version key (SHA1) via the Transitland Rest API: 

`requests.get(url='https://api.transit.land/api/v2/rest/feed_versions?api_key=xxx/0cf470c510a327cc5d5e0c597d2b69b448a6822e/download')`

The return I got is {'message': 'Invalid authentication credentials'}. Anything I can do here to fix this and download the GTFS zip?

Thanks!

**Comment by drewda** [comment, marked as answer]
Hi @shiqin-liu, please try moving your `?api_key` query parameter to the end of the URL. For example: 

````
https://api.transit.land/api/v2/rest/feed_versions/0cf470c510a327cc5d5e0c597d2b69b448a6822e/download?api_key=xxx
````

**Reply by shiqin-liu** [reply]
Thanks for the quick response! This works!
